lib/themes/dosomething/paraneue_dosomething/js/signup/templates/takeover.tpl.html

```diff
@@ -3,32 +3,16 @@
   <p><%= caption %></p>
 
   <div class="takeover__campaigns">
-     <ul class="form-actions -inline -padded takeover__campaign">
+    <ul class="form-actions -inline -padded takeover__campaign">
         <li class="takeover__campaign-item">
-          <h3>Thumb wars</h3>
-          <p>Description klSJHDADSF ASJDLA KASJDF UWE a lkjsdfhasdkfjh wudahksjdhf suew gkjak utgt akslkguihwh vjaskdjhfau wgag skauwgakja sdf askjdhfluweg akjhsdlkuyfwl </p>
+          <% for (var i=0; i < campaigns.length; i++){ %>
```

Comment:
I think you might need to include the whole `<ul></ul>` within the loop so like each campaign gets wrapped in the `ul` in the same way that we have it hardcoded